7-React/src/components/TodoItem/TodoItem.driver.tsx

```diff
@@ -0,0 +1,101 @@
+import TodoItem from "./TodoItem";
+import { fireEvent, render, RenderResult } from "@testing-library/react";
+import dataHooks from "./data-hooks";
+import React from "react";
+
+class TodoItemDriver {
+  private wrapper?: RenderResult;
+  private todoDescription: string = "";
+  private deleteTaskMock: jest.Mock = jest.fn();
+  private editTextMock: jest.Mock = jest.fn();
+  private toggleDone: jest.Mock = jest.fn();
+
+  private editBtn = () => {
+    const element = this.wrapper?.queryByTestId(dataHooks.editBtn);
+    return {
+      editBtnExists: () => {
+        return element ? true : false;
+      },
+      editBtnClick: () => {
+        if (element) {
```

Comment:
if the element doesn't exist, nothing will happen. this could be problematic if someone else will use this driver and expect the click to happen but no indication is given

**tl;dr** throw an error?